app/controllers/worldwide_organisations_controller.rb

```diff
@@ -23,8 +23,36 @@ def show
     end
   end
 
+  def show_b_variant
+    @world_location = WorldLocation.with_translations(I18n.locale).find(params[:world_location_id])
+    @embassy_data = embassy_test_data(params[:id])
+    @primary_role = primary_role
+    @other_roles = ([secondary_role] + office_roles).compact
+
+    set_meta_description(@worldwide_organisation.summary)
+    set_slimmer_organisations_header([@worldwide_organisation] + @worldwide_organisation.sponsoring_organisations)
+
+    if show_b_variant? && @embassy_data
```

Comment:
So the `&& @embassy_data` is effectively checking if there's any data for this organisation in the YAML file? In the other controllers we used the `is_under_test?` method from the WorldwideABTestHelper. Might not make a huge difference here though